app/views/notes/show.html.erb

```diff
@@ -130,3 +141,51 @@
   </div>
 
 </div><!--/span-->
+
+
+<style type="text/css">
+  .hexbin-hexagon {
+    stroke: #000;
+    stroke-width: 1px;
+  }
+</style>
+
+<script type="text/javascript">
```

Comment:
See this for recommended way to pass values into the JS in a separate file: https://github.com/publiclab/plots2/issues/604